practice/src/css/components/text.css

```diff
@@ -1,34 +1,38 @@
 .text {
-  color: var(--dark-text-color);
-  font-weight: var(--font-weight-semibold);
-  font-family: var(--primary-font); 
+  font-family: var(--primary-font);
 }
 
 .main-heading {
   color: var(--secondary-text-color);
-  font-size: var(--font-size-xsmall);
-  line-height: 22.4px;
-}
-
-.secondary-heading {
-  font-size: var(--font-size-2xl);
-  line-height: 50.4px;
+  font-weight: var(--font-weight-regular);
+  font-size: var(-font-size-small);
+  line-height: var(--tertiary-line-height);
 }
 
 .main-title {
-  font-size: var(--font-size-lg);
-  line-height: var(--primary-line-height);
+  color: var(--secondary-text-color);
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-small);
+  line-height: 23px;
 }
 
 .secondary-title {
-  font-size: var(--font-size-xxsmall);
-  font-weight: var(--font-weight-light);
-  font-style: italic;
+  color: var(--dark-text-color);
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-3xl);
+  line-height: 51px;
 }
 
 .description {
-  color: var(--tertiary-text-color);
-  font-size: var(--font-size-xsmall);
+  font-size: var(--font-size-small);
   font-weight: var(--font-weight-regular);
   line-height: var(--primary-line-height);
+  color: var(--tertiary-text-color);
+}
+
+.service-topic-title {
```

Comment:
If you only use this style for the service section, please help move this style to service.css

If you use this style for others, please update class name